included/10.0.0/Dockerfile

```diff
@@ -0,0 +1,62 @@
+# WARNING: this file was autogenerated by generate-included-image.js
+# using
+#   npm run add:included -- 10.0.0 cypress/browsers:node16.14.2-slim-chrome100-ff99-edge
+#
+# build this image with command
+#   docker build -t cypress/included:10.0.0 .
+#
+FROM cypress/browsers:node16.14.2-slim-chrome100-ff99-edge
+
+# avoid too many progress messages
+# https://github.com/cypress-io/cypress/issues/1243
+ENV CI=1 \
+# disable shared memory X11 affecting Cypress v4 and Chrome
+# https://github.com/cypress-io/cypress-docker-images/issues/270
+  QT_X11_NO_MITSHM=1 \
+  _X11_NO_MITSHM=1 \
+  _MITSHM=0 \
+  # point Cypress at the /root/cache no matter what user account is used
+  # see https://on.cypress.io/caching
+  CYPRESS_CACHE_FOLDER=/root/.cache/Cypress \
+  # Allow projects to reference globally installed cypress
+   NODE_PATH=/usr/local/lib/node_modules
```

Comment:
⚠️ This ENV is new, since cypress.config.js now requires cypress, without this line the docker container would fail to run without the consumer npm installing cypress. This allow local node projects to require globally installed npm modules.